TeamCode/src/main/java/org/firstinspires/ftc/teamcode/robot_components/robot/Robot.java

```diff
@@ -123,6 +127,10 @@ public void setBackIntakePower(double power){
         intakePower = power;
         intakeMotorBack.setPower(intakePower);
     }
+    public void setSpinnerPower(double power){
```

Comment:
remove this and use duck spinner class (simpleDuckSpinner?) in tele instead- having two different ways to move the duck spinner is dangerous, it should all be in the same place.